main.tf

```diff
@@ -1,13 +1,14 @@
-data "azurerm_resource_group" "parent" {
-  name = var.resource_group_name
-}
+# Data sources should be removed, due to best practice guidance from core team.
+# data "azurerm_resource_group" "parent" {
+#   name = var.resource_group_name
+# }
```

Comment:
Can we remove the commented out lines since they're no longer used?

Reply:
yup, I can remove this.